docs/addons/amp-addon.md

```diff
@@ -0,0 +1,123 @@
+# Amazon Managed Service for Prometheus (AMP)) Amazon EKS Add-on
+
+Amazon Managed Service for Prometheus is a Prometheus-compatible monitoring and alerting service that makes it easy to monitor containerized applications and infrastructure at scale. This Addon deploys an AWS Distro for OpenTelemetry (ADOT) Collector for Amazon Managed Service for Prometheus (AMP) which receives OTLP metrics from the application and Prometheus metrics scraped from pods on the cluster and remote writes the metrics to AMP remote write endpoint. You can change the mode to Daemonset, StatefulSet, and Sidecar depending on your deployment strategy.
+
+This driver is not automatically installed when you first create a cluster, it must be added to the cluster in order to setup AMP for remote write of metrics.
+
+For more information on the driver, please review the [user guide](https://docs.aws.amazon.com/eks/latest/userguide/opentelemetry.html).
+
+## Prerequisites
+- `Certificate-Manager` Blueprints Addon addon.
+- `ADOT` EKS Blueprints Addon.
+
+## Usage
+
+This Addon can used with four different patterns :
+
+Pattern # 1 : Simple and Easy - Using all default property values.
```

Comment:
What does this pattern do? Creates a new workspace? It is an important detail for the readers.